src/actions/hearingEditor.js

```diff
@@ -157,6 +158,28 @@ export function addContact(contact, selectedContacts) {
   };
 }
 
+export function saveContact(contact) {
+  return (dispatch, getState) => {
+    const url = `/v1/contact_person/${contact.id}/`;
+    const contactInfo = omit(contact, ['id']);
+    return api
+      .put(getState(), url, contactInfo)
+      .then(checkResponseStatus)
+      .then(response => {
+        if (response.status === 400) {
+          notifyError('Tarkista yhteyshenkilön tiedot.');
```

Comment:
I think this should be something like "Sinulla ei ole oikeutta muokata yhteyshenkilöä." as this error comes when trying to edit person from a organization I don't have edit rights for

Reply:
Should we use the error message from the response to display instead ?

Reply:
fixed.